examples/WorkerServiceSDK/ConsoleAppWithApplicationInsights/ConsoleAppWithApplicationInsights/Program.cs

```diff
@@ -0,0 +1,85 @@
+﻿using Microsoft.ApplicationInsights;
+using Microsoft.ApplicationInsights.Channel;
+using Microsoft.ApplicationInsights.DataContracts;
+using Microsoft.ApplicationInsights.Extensibility;
+using Microsoft.ApplicationInsights.Extensibility.PerfCounterCollector.QuickPulse;
+using Microsoft.ApplicationInsights.WindowsServer.TelemetryChannel;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace ConsoleAppWithApplicationInsights
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            // Create the DI container.
+            IServiceCollection services = new ServiceCollection();
+
+            // Add or configure channel
+            services.AddSingleton<ITelemetryChannel>(new ServerTelemetryChannel() { StorageFolder = @"C:\temp\aisdkstorage" });
+
+            // Add custom TelemetryInitializer
+            services.AddSingleton<ITelemetryInitializer, MyCustomTelemetryInitializer>();
+
+            // Being a regular console app, there is no appsettings.json or configuration providers enabled by default.
+            // Hence instrumentation key must be specified here.
+            services.AddApplicationInsightsTelemetryWorkerService("0eb18de0-7268-473c-ae06-c87ca480d2d2");
```

Comment:
Is this key expected here?

Reply:
no. @TimothyMothra lets put "put_actual_ikey_here" thing here.